classes/search/cmsfield.php

```diff
@@ -40,53 +43,6 @@ class cmsfield extends \core_search\base_mod {
      */
     protected $defaultvalues = null;
 
```

Comment:
We can remove $cmsdata and $defaultvalues method parameters.

Reply:
Removed.